README.md

```diff
@@ -1,65 +1,90 @@
-# Conceptual Functional Equivalent Model
-This model is designed to be a simplified (conceptual) model of the National Water Model, which is functionally equivalent.
+# Conceptual Functional Equivalent (CFE) Model
+This model is designed to be a simplified (conceptual) model of the runoff scheme in the National Water Model.
 
-# CFE was written by Fred Ogden, and this repository includes a Basic Model Interface (BMI) to run the CFE
-There are actually multiple ways to run the CFE.
-1. As written by the original  author. This includes a full program to read and process atmospheric forcing data, print the model output and check for mass balance closure. This code can be run from the `original_author_code` directory.
-2. Through the BMI commands. This method will be used by the Next Generation U.S. National Water Model (Nextgen NWM). The `make_and_run_bmi.sh` script in the main directory does that for you through a main run file in the `./src` directory.
+CFE was written by Fred Ogden, and this repository includes an implementation of the [Basic Model Interface (BMI)](https://bmi.readthedocs.io/en/latest/).
+There are multiple ways to run CFE.
+1. As written by the original author.
+This includes a full program to read and process atmospheric forcing data, print the model output and check for mass balance closure. This code can be run from the [original_author_code](./original_author_code) directory. This code does not have a BMI implementation.
+2. Through the BMI commands. This method will be used by the Next Generation National Water Model (Nextgen NWM). See [Compiling BMI Code](#compiling-bmi-code)
 
-## Direct runoff 
+## Direct Runoff 
 The user has the option to pick a particular direct runoff (aka surface partitioning) method:
 1. Schaake function (configuration: `surface_partitioning_scheme=Schaake`)
-2. XinanJiang function (configuration: `surface_partitioning_scheme=Xinanjiang`). When using this runoff method the user must also include three parameters.
-### If XinanJiang is choosen these parameters need to be included in the configuration file:
-1. a_Xinanjiang_inflection_point_parameter
-2. b_Xinanjiang_shape_parameter
-3. x_Xinanjiang_shape_parameter 
-
-# Compiling this code to run examples with a "pseudo" or "mini" framework.
-## Read local forcing file
-The BMI functionality was developed as a standalone module in C. To compile this code the developer used these steps:
+2. XinanJiang function (configuration: `surface_partitioning_scheme=Xinanjiang`).
+If XinanJiang is choosen these parameters need to be included in the configuration file:
+      ```
+      a_Xinanjiang_inflection_point_parameter
+      b_Xinanjiang_shape_parameter
+      x_Xinanjiang_shape_parameter
+      ```
+## Compiling BMI Code
+There are three unique examples for running cfe-bmi within a *pseudo* framework
```

Comment:
I don't think pseudo framework will resonate with anyone who is not familiar with Nextgen.

Reply:
would _mock framework_ be better?  Idea is to highlight `main()` is not a genuine model-coupling framework (or perhaps _infrastructure_?), such as `ngen`, but just acts to represent one by calling the various bmi functions.